netbox/netbox/constants.py

```diff
@@ -186,7 +186,7 @@
         'url': 'tenancy:tenant_list',
     }),
     ('contact', {
-        'queryset': Contact.objects.prefetch_related('group', 'assignments'),
+        'queryset': Contact.objects.prefetch_related('group', 'assignments').annotate(assignment_count=count_related(ContactAssignment, 'contact')),
```

Comment:
Change is out of scope for #8553. I've asked the OP to submit a separate bug report for this.

Reply:
okay, I will add remove that and make a new PR when the new issue is opened.

Reply:
The second issue is #8577 FYI